ember-cli-build.js

```diff
@@ -56,6 +57,20 @@ module.exports = function () {
         'public/images/stroke-icons',
         'public/images/svg'
       ]
+    },
+    svgJar: {
+      optimizer: {
+        svgoModule: SVGO,
```

Comment:
Injecting the latest SVGO package to use it in SVGJar instead of the old version.